src/ui/components/Collections.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import Select from 'react-select';
+import axios from 'axios';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow
+} from '@material-ui/core';
+import ResourceCell from './ResourceCell';
+
+const Collections = () => {
+  const [data, setData] = useState({ collectionName: '', headers: [], data: [] });
+  const collections = [
+    { value: 'servers', label: 'Servers' },
+    { value: 'endpoints', label: 'Endpoints' },
+    { value: 'subscriptions', label: 'Subscriptions' },
+    { value: 'plandefinitions', label: 'PlanDefinitions' },
+    { value: 'logs', label: 'Logs' }
+  ];
+
+  const onSelect = selectedOption => {
+    axios.get(`http://localhost:3000/${selectedOption.value}`).then(response => {
+      setData({
+        collectionName: selectedOption.value,
+        headers: getHeaders(selectedOption.value),
+        data: response.data
+      });
+    });
+  };
+
+  const getHeaders = collectionName => {
```

Comment:
FHIR resource collections should have `fullUrl` too since that is now their "key"